products/paas/fastly.md

```diff
@@ -0,0 +1,7 @@
+# Fastly
+
+
+1. Make sure you have at least Shopware 6.4.11.0
```

Comment:
We should explain what the benefits are of using Fastly 🤔

Reply:
what are the benefits?